README.md

```diff
@@ -1,10 +1,35 @@
 # OPA Gatekeeper Library
 
-A community-owned library of policies for the OPA Gatekeeper project.
+A community-owned library of policies for the [OPA Gatekeeper project](https://open-policy-agent.github.io/gatekeeper/website/docs/).
 
 ## Usage
 
-Apply the `template.yaml` and `constraint.yaml` provided in each directory under `library/`
+### kustomize
+
+You can use [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) to install some or all of the templates alongside your own contraints.
+
+First, create a `kustomization.yaml` file:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- github.com/open-policy-agent/gatekeeper-library/library
+# You can optionally install a subset by specifying a subfolder, or specify a commit SHA
+# - github.com/open-policy-agent/gatekeeper-library/library/pod-security-policy?ref=0c82f402fb3594097a90d15215ae223267f5b955
+- constraints.yaml
+```
+
+Then define your constraints in a file called `constraints.yaml` in the same directory. Example constraints can be found in the "samples" folders.
```

Comment:
Could this potentially install the same constraints multiple times since the samples folders contain constraints as well?

Reply:
nvm, that kustomize file doesn't include the samples folder.